inithome/pyproject.toml

```diff
@@ -0,0 +1,80 @@
+[project]
+# https://packaging.python.org/en/latest/specifications/declaring-project-metadata/
+name = "rubin.nublado.inithome"
+description = "Home directory provisioner for Nublado."
+license = { file = "LICENSE" }
+readme = "README.md"
+keywords = ["rubin", "lsst"]
+# https://pypi.org/classifiers/
+classifiers = [
+    "Development Status :: 4 - Beta",
+    "License :: OSI Approved :: MIT License",
+    "Programming Language :: Python",
+    "Programming Language :: Python :: 3",
+    "Programming Language :: Python :: 3.11",
+    "Natural Language :: English",
+    "Operating System :: POSIX",
+    "Typing :: Typed",
+]
+requires-python = ">=3.11"
+dependencies = []
+dynamic = ["version"]
+
+[project.optional-dependencies]
+dev = [
+    # Testing
+    "pytest",
+    "pytest-cov",
+    "pytest-asyncio",
+    "pyfakefs",
+    # To work with higher-level nox/pyproject
+    "pydantic"
+]
```

Comment:
Since the output here is a Docker container rather than something that's pip-installable, I think this should use frozen dependencies like a service, and thus `requirements/dev.in` and so forth. It's total overkill, I know, but it maintains consistency between the controller and inithome so that we don't introduce yet a third pattern that's a hybrid between the pip-installable plugins and the things that generate a Docker file.

I assume "To work with higher-level nox/pyproject" is because the top-level pyproject.toml configures the Pydantic mypy plugin and therefore Pydantic has to be installed for mypy to run? If so, I'd say that explicitly; right now it's not clear what this is referring to.

Reply:
Easy stuff hammered out.

Reply:
requirements updated.